src/platforms/kotlin-multiplatform/config.yml

```diff
@@ -0,0 +1,9 @@
+title: Kotlin Multiplatform
+caseStyle: camelCase
+fallbackPlatform: java
+supportLevel: production
```

Comment:
What's the other option here? since it's not `production` ready.

Reply:
https://docs.sentry.io/contributing/platforms/#configuration

Seems like 'community' would be the other alternative but not sure how much that fits

Reply:
Good question, maybe the docs team can chime in here.